e2e/test/scenarios/admin/people/people.cy.spec.js

```diff
@@ -95,10 +95,24 @@ describe("scenarios > admin > people", () => {
 
       // should load the members when navigating to the group directly
       cy.visit(`/admin/people/groups/${DATA_GROUP}`);
+
       // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
       cy.findByText("2 members");
 
-      removeUserFromGroup(noCollectionUserName);
+      cy.findByRole("list", { name: "admin-list-items" })
+        .findByRole("link", { name: /people/i })
+        .click();
+
+      showUserOptions(noCollectionUserName);
+
+      popover().findByText("Deactivate user").click();
+
+      clickButton("Deactivate");
+
+      cy.findByRole("link", { name: /group/i }).click();
+
+      cy.findByRole("table").findByRole("link", { name: /data/i }).click();
+
```

Comment:
We've already covered deactivating a user via UI. No need to repeat that here.

Why don't we instead:
1. use API to deactivate the user, 
2. navigate to the group directly
3. assert on the number of users

Reply:
There was an issue I found when testing where if you had already loaded the group details, navigated away, archived a user and then navigated back, it would display the wrong count. That's why I added all that in there

Reply:
I meant - we can deactivate a user before we start this test.

Reply:
If you look at the changes to `people.js`, it appears that we also fixed a caching bug with the combination of using the button to delete members and seeing the updated number of members, so the test is actually necessary to test that edge case.